packages/circuit-ui/components/ImageInput/ImageInput.tsx

```diff
@@ -131,24 +144,28 @@ const invalidLabelStyles = ({ theme, invalid }: StyledLabelProps) =>
       left: 0%;
       width: 100%;
       height: 100%;
-      border-radius: 12px;
+      border-radius: ${theme.borderRadius[borderRadius]};
```

Comment:
👀 must have forgotten to update this one

Reply:
Looks like it. I'll fix it.